.github/workflows/ci.yml

```diff
@@ -554,6 +554,14 @@ jobs:
             python_ver: "3.13"
             openimageio_ver: main
             setenvs: export LLVMBREWVER="@19"
+          - desc: MacOS-15-ARM aclang16/C++17/py3.13 llvm21 oiio-main
+            runner: macos-15
+            nametag: macos15-arm-py313
+            cc_compiler: clang
+            cxx_compiler: clang++
+            cxx_std: 17
+            python_ver: "3.13"
+            openimageio_ver: main
```

Comment:
Should this be left in? If LLVM 21 compat is claimed it wouldn't be too bad if there is a CI testing it I guess 😊

Reply:
I don't think you need to add yet another Mac test -- just change the existing macos-15 test we already have to use the latest LLVM (by removing the `LLVMBREWVER` reference).

Also, in the Linux "bleeding edge" test, we should change the `llvm_action_ver: "18.1.7"` to use the 21.1 (you will probably also need to change build-steps.yml to bump KyleMayes/install-llvm-action to the newer version that knows about llvm 21.